.github/workflows/validate.yml

```
@@ -21,7 +21,7 @@ jobs:
   linters:
     strategy:
       matrix:
-        go-version: [1.21.x]
+        go-version: [1.21.x, 1.22.x]
```

```suggestion
        go-version: [1.22.x]
```
Just latest should be fine for this.